doc/source/math/resizing_ops.rst

```diff
@@ -0,0 +1,356 @@
+.. _resizing_ops:
+
+##################
+Resizing Operators
+##################
+
+
+Introduction
+============
+Resizing can be described in a continuous as well as in a fully discrete setting. In the
```

Comment:
This paragraph does not (at least to me) fully describe what you mean by resizing. "Changing the domain" is abit vague. Change in what way? It would be good to be abit more clear here since this is the most important paragraph.